pa11y_scripts/.crt-view-date.json

```diff
@@ -0,0 +1,27 @@
+{
+    "urls": [
+        {
+            "useIncognitoBrowserContext": true,
+            "url": "http://127.0.0.1:8000/form/view/",
+            "actions": [
+                "wait for url to be http://127.0.0.1:8000/accounts/login/?next=/form/view/",
+                "set field #id_username to pa11y_tester",
+                "set field #id_password to imposing40Karl5monomial",
+                "click element #login",
+                "wait for path to be /form/view/",
+                "screen capture pa11y-screenshots/crt-view-date-step1.png",
+                "click element #date",
+                "screen capture pa11y-screenshots/crt-view-date-step2.png",
+                "wait for element #id_create_date_start to be visible",
+                "set field #id_create_date_start to 2020-01-01",
+                "screen capture pa11y-screenshots/crt-view-date-step3.png",
+                "wait for element #id_create_date_end to be visible",
+                "set field #id_create_date_end to 2020-07-01",
+                "screen capture pa11y-screenshots/crt-view-date-step4.png",
+                "click #id_apply_dates",
+                "wait for path to be /form/view/"
+            ],
+            "screenCapture": "pa11y-screenshots/crt-sort-view-date-final.png"
```

Comment:
:raised_hands:

Reply:
revert as in all the way back? or just remove screencaptures?

Reply:
Two suggestions, remove the screen capture in the middle (it doesn't seem useful, but correct me if i'm wrong) and use `id_status` instead of `assigned_to` as in the original.

Reply:
i'm afraid any change is breaking pa11y run without timeouts. I recommend leaving it as is.

Reply:
Wait, so removing the screen capture makes it time out? :thinking: that is indicative of a race condition (since doing a screenshot takes up some time, and presumably this pause helps subsequent actions execute). 

I think we need to address pa11y fixes more holistically -- among other things, these changes will be easier to review -- sooo I'm not sure if these changes belong to this PR now.

Are there any changes in this PR that directly impact this test? If not, can we revert this whole file for now?

Reply:
If we are fixing all of pa11y scripts, it doesn't belong here. Static a11y tests pass for this PR, so pa11y script not passing is not a direct impact. I recommend we leave the file pa11y_scripts/.crt-view-date.json in there and not link it in "test:ally" command in package.json, But, if you say remove it, i'm okay with it.